cmd/config/notify/parse.go

```diff
@@ -275,12 +269,24 @@ func RegisterNotificationTargets(cfg config.Config, doneCh <-chan struct{}, tran
 		if err != nil {
 			return nil, err
 		}
+		if err := targetList.Add(newTarget); err != nil {
+			return nil, err
+		}
 		if test {
 			newTarget.Close()
 			continue
 		}
-		if err := targetList.Add(newTarget); err != nil {
-			return nil, err
+	}
+
+	if test {
+		// Verify if user is trying to disable already configured
+		// notification targets, based on their target IDs
```

Comment:
len of targetIDs will be 0 always, if the globalNotificationSys hasn't initialized yet

Reply:
verified this with print statements

Reply:
If it's not initialized then bad luck we can't do much.